course/src/main/java/org/openedx/course/presentation/container/CourseContainerFragment.kt

```diff
@@ -489,9 +518,128 @@ fun DashboardPager(
     }
 }
 
+@Composable
+private fun CourseAccessErrorView(
+    viewModel: CourseContainerViewModel?,
+    accessError: CourseAccessError?,
```

Comment:
It's already a pretty big class; what about moving these views to a separate file?

Reply:
Will be done in the marketplace repo first.